### PR TITLE
PHOENIX-6543 de-flake AuditLoggingIT

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/AuditLoggingIT.java [hunk 1]
```diff
@@ -65,91 +68,117 @@ public void testLoggingSelect() throws Exception {
...
-            conn.close();
+            int retryCount=10;
+            while (true) {
+                try {
+                    ResultSet rs2 = stmt.executeQuery(getLogsQuery);
+                    assertTrue(rs2.next());
+                    assertEquals(rs2.getString(7), selectQuery);
+                    assertFalse(rs2.next());
+                    break;
+                } catch (AssertionFailedError e) {
```
Comment (richardantal) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/AuditLoggingIT.java:90]: Shouldn't we catch AssertionError instead of AssertionFailedError?

Comment (stoty) [reply phoenix-core/src/it/java/org/apache/phoenix/end2end/AuditLoggingIT.java]: Thanks, you are right.
Fixed.

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 31s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  30m  6s |  master passed  |
| +0 |  hbaserecompile  |  39m 37s |  HBase recompiled.  |
| +1 :green_heart: |  compile  |   1m 28s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 18s |  master passed  |
| +1 :green_heart: |  javadoc  |   1m 16s |  master passed  |
| +0 :ok: |  spotbugs  |   4m 18s |  phoenix-core in master has 972 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  23m 47s |  the patch passed  |
| +0 |  hbaserecompile  |  37m 50s |  HBase recompiled.  |
| +1 :green_heart: |  compile  |   1m 37s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 37s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   1m 20s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  javadoc  |   1m 10s |  the patch passed  |
| +1 :green_heart: |  spotbugs  |   4m 31s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 158m 47s |  phoenix-core in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 52s |  The patch does not generate ASF License warnings.  |
|  |   | 255m 43s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | phoenix.end2end.index.IndexMetadataIT |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1302/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix/pull/1302 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs hbaserebuild hbaseanti checkstyle compile |
| uname | Linux 58d61de4c478 4.15.0-143-generic #147-Ubuntu SMP Wed Apr 14 16:10:11 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-personality.sh |
| git revision | master / e84232b |
| Default Java | Private Build-1.8.0_242-8u242-b08-0ubuntu3~16.04-b08 |
| unit | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1302/1/artifact/yetus-general-check/output/patch-unit-phoenix-core.txt |
|  Test Results | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1302/1/testReport/ |
| Max. process+thread count | 6223 (vs. ulimit of 30000) |
| modules | C: phoenix-core U: phoenix-core |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1302/1/console |
| versions | git=2.7.4 maven=3.3.9 spotbugs=4.1.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   4m 13s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  20m 57s |  master passed  |
| +0 |  hbaserecompile  |  27m 43s |  HBase recompiled.  |
| +1 :green_heart: |  compile  |   1m  3s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 55s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 51s |  master passed  |
| +0 :ok: |  spotbugs  |   3m  7s |  phoenix-core in master has 972 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  13m 28s |  the patch passed  |
| +0 |  hbaserecompile  |  23m 26s |  HBase recompiled.  |
| +1 :green_heart: |  compile  |   1m  2s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m  2s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 56s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  javadoc  |   0m 51s |  the patch passed  |
| +1 :green_heart: |  spotbugs  |   3m 15s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 112m 27s |  phoenix-core in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 43s |  The patch does not generate ASF License warnings.  |
|  |   | 180m 47s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1302/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix/pull/1302 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs hbaserebuild hbaseanti checkstyle compile |
| uname | Linux af669b73ec5f 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-personality.sh |
| git revision | master / c55b472 |
| Default Java | Private Build-1.8.0_242-8u242-b08-0ubuntu3~16.04-b08 |
| unit | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1302/2/artifact/yetus-general-check/output/patch-unit-phoenix-core.txt |
|  Test Results | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1302/2/testReport/ |
| Max. process+thread count | 15010 (vs. ulimit of 30000) |
| modules | C: phoenix-core U: phoenix-core |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1302/2/console |
| versions | git=2.7.4 maven=3.3.9 spotbugs=4.1.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by richardantal — APPROVED** [review]
LGTM

**Pull request closed without merging**